### PR TITLE
1.0.3

### LINKED ISSUE #9: Should have a config option to check for extraneous keys in json
This should default to checking!

In case a user misspells an optional key like `errorExpected` -> `expectedError`

### LINKED ISSUE #10: Don't log the result/error
This can cause timeouts for students

### PR DESCRIPTION
**Pull request by braxtonhall**
resolves #9 
resolves #10

### CODE REVIEW

File: src/testFolder.ts [hunk 1]
```diff
@@ -31,7 +31,8 @@ function testFolder<I, O, E>(suiteName: string,
                     const result = await target(test.input);
                     if (test.errorExpected) {
                         // if we want an error just reject again
-                        return Promise.reject(new Error(`Expected an error but instead resolved or returned with ${result}`));
```
Comment (falkirks) [inline src/testFolder.ts:34]: I would argue that this is a backwards incompatiable change and we shouldn't tag it with `1.0.3`. But also I don't really think the students will care.

**Review by falkirks — COMMENTED** [review]
🚢

File: src/testFolder.ts (hunk 1 above)
Comment (braxtonhall) [reply src/testFolder.ts:34]: Yeah but people were getting timeouts because of it :(

File: src/Options.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ function getDefaultOptions<I, O, E>(): TestFolderOptions<I, O, E> {
     return {
         assertOnError,
         assertOnResult,
+        checkForExcessKeys: true,
```
Comment (braxtonhall) [inline src/Options.ts:16]: Now that I think of it, _this_ is the backwards incompatible change. Their tests that used to pass now won't even run on the bot

**Pull request merged**